docs/source/gs-processing/usage/distributed-processing-setup.rst

```diff
@@ -104,13 +104,65 @@ the following to build the SageMaker image:
     bash docker/build_gsprocessing_image.sh --environment sagemaker
 
 The above will use the SageMaker-specific Dockerfile of the latest available GSProcessing version,
-build an image and tag it as ``graphstorm-processing-sagemaker:${VERSION}`` where
+build an image and tag it as ``graphstorm-processing-sagemaker:${VERSION}-x86_64`` where
 ``${VERSION}`` will take be the latest available GSProcessing version (e.g. ``0.2.1``).
 
 The script also supports other arguments to customize the image name,
 tag and other aspects of the build. See ``bash docker/build_gsprocessing_image.sh --help``
 for more information.
 
+Support for arm64 architecture
+------------------------------
+
+You might have noticed that we include the image's architecture, ``x86_64`` in the image name.
+For EMR Serverless images, it is possible to build images that support ``arm64`` instances,
+which can lead to improved runtime and cost compared to ``x86_64``. To build ``arm64`` images
```

Comment:
Can user build arm64 images on arm64 servers?
Qemu emulation may be too complex.

Reply:
They canI can add that as an alternative.